contracts/seed/Seed.sol

```diff
@@ -164,17 +164,19 @@ contract Seed {
         seedRemainder = seedRemainder.sub(_seedAmount);
         //  fundingAmount is an amount of fundingTokens required to buy _seedAmount of SeedTokens
         uint256 fundingAmount = (_seedAmount.mul(price)).div(PCT_BASE);
+        // Funding Token balancec of this contract;
+        uint256 seedContractFundingTokenBal = fundingToken.balanceOf(address(this));
```

Comment:
why such a long name? we can call it just `fundingBalance`

Reply:
renamed